Modules/modulesHelper.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -e ;# exit on error
+set -u ;# exit when using undeclared variable
+#set -x ;# debugging
+
+DONOR=SkeletonDPL
+DONOR_LC=skeleton_dpl
+DONOR_TASK=SkeletonTaskDPL
+DONOR_TASK_INCLUDE_GUARD=QC_MODULE_EXAMPLE_EXAMPLETASKDPL_H
```

Comment:
DONOR_TASK_INCLUDE_GUARD could be generated automatically with the DONOR_TASK : `QC_MODULE_${DONOR}_${DONOR_TASK}_H`

Reply:
And of course, the guard should be updated in SkeletonTaskDPL.h to match this... (and it will make more sense).

Reply:
I can see that include guards could be fixed also in other modules by the occasion, so I will do it in a separate PR.

Reply:
Actually, it needs to be ```QC_MODULE_`echo ${DONOR} | tr a-z A-Z`_`echo ${DONOR_TASK} | tr a-z A-Z`_H``` if we want to preserve uppercase convention

Reply:
> I can see that include guards could be fixed also in other modules by the occasion, so I will do it in a separate PR.

Good, thank you.

Reply:
> Actually, it needs to be `` QC_MODULE_`echo ${DONOR} | tr a-z A-Z`_`echo ${DONOR_TASK} | tr a-z A-Z`_H `` if we want to preserve uppercase convention

Correct